content/en/benchmark.md

```diff
@@ -0,0 +1,128 @@
+
+Transonic is a pure python package. It is one of the actively growing libraries which is used to increase the speed of our codes. Here, we are operating with two kinds of accelerators in the backend jit and boost to accelerate our code. Transonic uses four different types of accelerators in the backend Pythran, Cython, Numba, and Python. Presently, we have implemented Jit and boost accelerators, in the next iteration, we will be using Cython, with different types of backends.
+
+#### Transonic - JIT
+
+We used Transonic jit to accelerate our code. To implement this we used Pythran (Ahead-Of-Time) compiler in Just-In-Time mode. This algorithm took 0.01 seconds for input datasets with 16 points, 0.05 seconds for input datasets with 32 points and 0.17 seconds for input datasets with 64 points.
+ 
+#### Transonic - Boost: Pythran
+
+Pythran is an Ahead Of Time compiler. This module takes a Python file as an input then converts it into some kind of interface which makes the code faster. Transonic's boost decorator replaces the python function with the pythranized function. This algorithm took 10.37 seconds for input data of 16 points, 41.88 seconds for input data with 32 points and 160.15 seconds for input data with 64 points.
+
+### Numba
+
+Numba converts the Python files to optimized machine code during runtime. The major advantage is after compiling codes using Numba, the speed of code becomes approachable to C or FORTRAN. Numba caches all the compiled functions into a file system for future use of the same functions. We used the jit decorated functions to call another jit decorated function while implementing our code. It is advisable to add the jit decorated function to other functions too, so as to prevent it from generating slower outputs. This algorithm took 0.89 seconds for input data with 16 points, 0.05 seconds for input data with 32 points and 0.17 seconds for input data with 64 points.
+
+## Results 
+
+Table values represent the time taken by each algorithm to run, in respected datasets for 5 iterations.
+
+<html>
+<table>
+ <tr>
+  <td></td>
+  <td><b>Input 16</b></td>
+  <td><b>Input 32</b></td>
+  <td><b>Input 64</b></td>
+ </tr>
+ <tr>
+  <td><b>Pythran</b></td>
+  <td>10.37</td>
+  <td>41.88</td>
+  <td>160.15</td>
+ </td>
+ <tr>
+  <td><b>Numba</b></td>
+  <td>0.89</td>
+  <td>0.05</td>
+  <td>o.17</td>
+ </tr>
+ <tr>
+  <td><b>Pythran Naive</b></td>
+  <td>0.01</td>
+  <td>0.05</td>
+  <td>0.17</td>
+ </tr>
+ <tr>
+  <td><b>NumPy</b></td>
+  <td>10.15</td>
+  <td>45.2</td>
+  <td>161.01</td>
+ </tr>
+ <tr>
+  <td><b>Pure NumPy</b></td>
+  <td>10.56</td>
+  <td>41.59</td>
+  <td>169.4</td>
+ </tr>
+</table>
+</html>
+
+## Code Folders
+
+* These codes are originally taken from [here](https://github.com/paugier/nbabel)
+* Benchmark code: [here](https://github.com/khushi-411/numpy-benchmarks/blob/0.3/python/benchmark-2.py)
+* Graph: [here](https://github.com/khushi-411/numpy-benchmarks/blob/0.3/python/plot-modified-1.py)
```

Comment:
Linking to your own repo is fine for now. We can make it a separate NumPy repo or put the scripts in an existing repo somewhere, when you are closer to done with this PR.

Reply:
Sure definitely, these are just sample codes. I feel that these need to be improved, and once it's finalized, we can create a repo or update somewhere in the existing repo as you'll wish. Thanks!